README.md

```diff
@@ -11,7 +11,22 @@ gem 'redis-rack-cache'
 
 ## Usage
 
-If you are using redis-store with Rails, consider using the [redis-rails gem](https://github.com/redis-store/redis-rails) instead. For standalone usage:
+If you are using redis-store with Rails, consider using the [redis-rails gem](https://github.com/redis-store/redis-rails) instead.
+However, configuration can be done a such:
+
+```ruby
+# config/application.rb
+module MyApplication
+  class Application < Rails::Application
+    config.action_dispatch.rack_cache = {
+      metastore: ::Rack::Cache::EntityStore::Redis.new('redis://localhost:6379/0/metastore', default_ttl: 10.days.to_i),
```

Comment:
Neither approaches are actually working. I've tested it locally by pointing the Gemfile to my git repository, and whether I use my notation or the one recommended, the TTL keeps falling back to one year. Meaning the information is not passed down.